lib/src/main/java/co/nimblehq/alpine/lib/AlpineLib.kt

```diff
@@ -0,0 +1,55 @@
+package co.nimblehq.alpine.lib
+
+import android.content.Context
+import android.nfc.Tag
+import co.nimblehq.alpine.lib.model.CameraImage
+import co.nimblehq.alpine.lib.model.MrzInfo
+import co.nimblehq.alpine.lib.mrz.MrzProcessor
+import co.nimblehq.alpine.lib.mrz.MrzProcessorImpl
+import co.nimblehq.alpine.lib.mrz.MrzProcessorResultListener
+import co.nimblehq.alpine.lib.nfc.ISO_DEP_TIMEOUT_IN_MILLIS
+import co.nimblehq.alpine.lib.nfc.NfcReader
+import co.nimblehq.alpine.lib.nfc.NfcReaderImpl
+import com.google.mlkit.vision.common.InputImage
+
+object AlpineLib {
+
+    private val nfcReader: NfcReader = NfcReaderImpl()
+    private val mrzProcessor: MrzProcessor = MrzProcessorImpl()
```

Comment:
Why did we create this class? Doesn't it mean the end user will be locked into using _both_ `MrzProcessor` and `NfcReader`? 🤔 What if they only need one of the classes? 🤔

Reply:
@ryan-conway To get rid of `companion object` in the `Interface` class. Also when calling the lib, it's easy to understand that we are calling `AlpineLib` to process the image and read NFC from our lib 🙏